src/itinerary/story.tsx

```diff
@@ -25,6 +25,33 @@ const places = [
     stepAriaLabel: 'Dropoff location',
     mainLabel: 'Gare Bordeaux Saint-Jean',
     subLabel: 'Bordeaux',
+    href: '#',
+    connectionLabel: 'Change of vehicle',
+  },
+]
+
+const places2 = [
+  {
+    time: '15:00',
+    isoDate: '2017-12-11T15:00',
+    stepAriaLabel: 'Dropoff location',
```

Comment:
It's just a content detail but the second place of a segment is a **Pickup location** instead of a **Dropoff location**